resources/shaders/pp_dof_bokeh_post_filter.hlsl

```diff
@@ -46,29 +50,22 @@ void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
 		[unroll]
 		for (int x = -SampleRadius; x <= SampleRadius; ++x)
 		{
-			near_color += source_near[screen_coord + float2(x, y)].rgb;
-			far_color += source_far[screen_coord + float2(x, y)].rgb;
+			near_color += source_near.SampleLevel(s0, (screen_coord + float2(x, y)) / screen_size, 0).rgb;//[screen_coord + float2(x, y)].rgb;
+			far_color += source_far.SampleLevel(s0, (screen_coord + float2(x, y)) / screen_size, 0).rgb;
 		}
 	}
 
 	near_color /= float(SampleDiameter * SampleDiameter);
 	far_color /= float(SampleDiameter * SampleDiameter);
-/*
-	float3 near_color = 
-		source_near.SampleLevel(s0, uv + offset.xy, 0).rgb +
-		source_near.SampleLevel(s0, uv + offset.zy, 0).rgb +
-		source_near.SampleLevel(s0, uv + offset.xw, 0).rgb +
-		source_near.SampleLevel(s0, uv + offset.zw, 0).rgb;
-	
-	near_color *= 0.25f;
 
-	float3 far_color = 
-		source_far.SampleLevel(s0, uv + offset.xy, 0).rgb +
-		source_far.SampleLevel(s0, uv + offset.zy, 0).rgb +
-		source_far.SampleLevel(s0, uv + offset.xw, 0).rgb +
-		source_far.SampleLevel(s0, uv + offset.zw, 0).rgb;
+	//float3 near_color = 
```

Comment:
Either get rid of commented code or add a comment that tells us why you keep that commented code there.